app/controllers/home_controller.rb

```diff
@@ -0,0 +1,4 @@
+class HomeController < ApplicationController
+  def index
```

Comment:
if there is nothing to add in action use this formate to write it def index; end